app/Platform/Actions/WriteGameSortTitleFromGameTitleAction.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Actions;
+
+use App\Models\Game;
+
+class WriteGameSortTitleFromGameTitleAction
+{
+    public function execute(Game $game, string $originalTitle, bool $shouldRespectCustomSortTitle = true): ?string
```

Comment:
The action accepts the game and its original title value from before the action was called. This lets us perform a check within the action to determine if a custom sort title override is in place so we can not accidentally override that custom value.